burn-import/onnx-tests/tests/transpose/transpose.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+# used to generate model: transpose.onnx
+
+import torch
+import torch.nn as nn
+
+
+class Model(nn.Module):
+    def __init__(self):
+        super(Model, self).__init__()
+
+    def forward(self, x):
+        x = x.transpose(0, 1)
+        return x
+
+
+def main():
+
+    # Set seed for reproducibility
+    torch.manual_seed(42)
+
+    torch.set_printoptions(precision=8)
+
+    # Export to onnx
+    model = Model()
+    model.eval()
+    device = torch.device("cpu")
+
+    file_name = "transpose.onnx"
+    test_input = torch.randn(2, 3, device=device)
+    torch.onnx.export(model, test_input, file_name,
+                      verbose=False, opset_version=16)
+
+    print("Finished exporting model to {}".format(file_name))
+
+    # Output some test data for use in the test
+    print("Test input data: {}".format(test_input))
+    print("Test input data shape: {}".format(test_input.shape))
+    output = model.forward(test_input)
+    print("Test output data shape: {}".format(output.shape))
+    print("Test output data: {}".format(output))
```

Comment:
This is starting to become repetitive. Probably not in this PR, but I think it would be a good idea to add a utility Python module that we could import into each script.

Reply:
Sure. It seems like a good idea.

Reply:
Yeah I agree with code duplication. I think we could isolate setting the seed, setting the device, setting the printing precision and exporting the model to ONNX into a Python module, so that we can update those based on PyTorch releases. The remaining isn't generic enough to be shared.